### PR TITLE
Docs: contributing guidelines

### PR DESCRIPTION
**Pull request by arathibiju**
# PR Template

## Description
I have added a contributing.md file within the root directory, which has the contributing guidelines that people can refer to. This format has been agreed on by all team members during a team meeting. Edit : Closes issue #4 

## How has this been tested?
N/A

## Checklist
*(Leave blank if not applicable)*

[ ] My code follows the style guidelines of this project
[ ] I have performed a self-review of my own code
[ ] I have commented my code, particularly in hard-to-understand areas
[ ] I have made corresponding changes to the documentation
[ ] My changes generate no new warnings
[ ] I have added tests that prove my fix is effective or that my feature works
[ ] New and existing unit tests pass locally with my changes
[ ] Any dependent changes have been merged and published in downstream modules
[ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Contributing to the SE701 Team 1 Project
+
+Thank you for investing your time in contributing to our project! 
+
+## Code of Conduct
+This project and everyone participating in it is follows our Code of Conduct (found in the root of this repository). By participating, you are also expected to uphold its standards and are subject to its enforcement guidelines.
```
Comment (JafarMaash) [inline CONTRIBUTING.md:6]: "participating in it _is_ follows" 
Nitpick, but is the "is" correct grammar there?

**Review by JafarMaash — CHANGES_REQUESTED** [review]
See comment

File: CONTRIBUTING.md (hunk 1 above)
Comment (arathibiju) [reply CONTRIBUTING.md:6]: Ah yes you're right! I'll fix that up, thank you

Comment (arathibiju) [reply CONTRIBUTING.md]: fixed now

**Review by KirstyGong — APPROVED** [review]

**Review by dshah1901 — APPROVED** [review]

**Review by JafarMaash — APPROVED** [review]

**Review by scorpionknifes — APPROVED** [review]

**Pull request merged**